docs/reference/objects/product/index.md

```diff
@@ -85,10 +85,10 @@ The duration of the product including duration unit e.g. `5 nights`.
 
 ## `product.experience_dates`
 {: .d-inline-block }
-[ExperienceDate]({% link docs/reference/objects/product/experience_date.md %})
+[Experience Date]({% link docs/reference/objects/product/experience_date.md %})
```

Comment:
Actually, we don't return this on the drop so we should delete this!

Reply:
I've remembered why, I keep forgetting this. Removed.